src/components/CCPDashboard/CCPDashboardPage.tsx

```diff
@@ -33,15 +52,16 @@ interface TabPanelProps {
   value: CCPDashboardTabOptions;
   className?: string;
 }
+type LocationState = { userUpdatedPatientId: string | null };
 
 const useStyles = makeStyles({
   root: {
     minHeight: '100vh',
     background: Colours.BackgroundGray,
   },
   tabPanel: {
-    paddingLeft: '30px',
-    paddingRight: '30px',
+    paddingLeft: '56px',
+    paddingRight: '56px',
```

Comment:
let's revert this - it's a styling fix i pushed directly to staging/master that i'd like to keep for now
```suggestion
    paddingLeft: '30px',
    paddingRight: '30px',
```